src/UXClient/Components/ScatterPlot/ScatterPlot.ts

```diff
@@ -101,6 +100,9 @@ class ScatterPlot extends ChartVisualizationComponent {
             this.pointWrapper = this.g.append("g")
                 .classed("tsi-pointWrapper", true);
 
+            this.lineWrapper = this.g.append("g")
```

Comment:
I think this line and line 100 should be swapped, so that the points sit on top of lines - unless there's some other reason to have them in this order that I'm not thinking of?

Here's with current order:
![image](https://user-images.githubusercontent.com/35545215/98431126-b7a9a200-2067-11eb-934d-609d8efe3350.png)


And here's with them swapped:
![image](https://user-images.githubusercontent.com/35545215/98431180-da3bbb00-2067-11eb-8a18-3c814cabe61c.png)

Reply:
Made that change, thanks for taking a look : )